tank/resources/providers/digitalocean/main.tf

```diff
@@ -3,13 +3,16 @@
 variable "token" {}
 variable "pvt_key" {}
 variable "ssh_fingerprint" {}
+variable "scripts_path" {}
 
 # test case-specific settings
 variable "blockchain_name" {}
 
 # run-specific settings
 variable "setup_id" {}
-
+variable "tank_region" {
```

Comment:
Please, don't introduce the variable. The region values should be passed via templating, not variables.